web/pages/api/v0/bets.ts

```diff
@@ -1,133 +1,4 @@
-    order: z.enum(['asc', 'desc']).optional(),
-  })
-  .strict()
-
-const getContractId = async (params: z.infer<typeof queryParams>) => {
-  if (params.contractId) {
-    return params.contractId
-  }
-  const slug = params.contractSlug ?? params.market
-  if (slug) {
-    const contract = await getContractFromSlug(slug, db)
-    if (contract) {
-      return contract.id
-    } else {
-      throw new Error('Contract not found.')
-    }
-  }
-}
-
-const getUserId = async (params: z.infer<typeof queryParams>) => {
-  if (params.userId) {
-    return params.userId
-  }
-  if (params.username) {
-    const user = await getUserByUsername(params.username)
-    if (user) {
-      return user.id
-    } else {
-      throw new Error('User not found.')
-    }
-  }
-}
-
-// mqp: this pagination approach is technically incorrect if multiple bets
-// have the exact same createdTime, but that's very unlikely
-
-const getBeforeTime = async (params: z.infer<typeof queryParams>) => {
-  if (params.before) {
-    const beforeBet = await getBet(params.before)
-    if (beforeBet == null) {
-      throw new Error('Bet specified in before parameter not found.')
-    }
-    return beforeBet.createdTime
-  } else {
-    return undefined
-  }
-}
-
-const getAfterTime = async (params: z.infer<typeof queryParams>) => {
-  if (params.after) {
-    const afterBet = await getBet(params.after)
-    if (afterBet == null) {
-      throw new Error('Bet specified in after parameter not found.')
-    }
-    return afterBet.createdTime
-  } else {
-    return undefined
-  }
-}
-
-export default async function handler(
-  req: NextApiRequest,
-  res: NextApiResponse<Bet[] | ValidationError | ApiError>
-) {
-  await applyCorsHeaders(req, res)
-
-  let params: z.infer<typeof queryParams>
-  try {
-    params = validate(queryParams, req.query)
-  } catch (e) {
-    if (e instanceof ValidationError) {
-      return res.status(400).json(e)
-    }
-    console.error(`Unknown error during validation: ${e}`)
-    return res.status(500).json({ error: 'Unknown error during validation' })
-  }
-
-  const { limit, order } = params
-  try {
-    const [userId, contractId, beforeTime, afterTime] = await Promise.all([
-      getUserId(params),
-      getContractId(params),
-      getBeforeTime(params),
-      getAfterTime(params),
-    ])
-    const opts: BetFilter = {
-      userId,
-      contractId,
-      beforeTime,
-      afterTime,
-      limit,
-      order,
-    }
-    if (params.kinds === 'open-limit') {
-      opts.isOpenLimitOrder = true
-    }
-    const bets = contractId
-      ? await getBets(db, opts)
-      : await getPublicBets(opts)
-    res.setHeader('Cache-Control', 'max-age=15, public')
-    return res.status(200).json(bets)
-  } catch (e) {
-    console.error(`Error while fetching bets: ${e}`)
-    return res.status(500).json({ error: 'Error while fetching bets: ' + e })
-  }
-}
+const handler = nextHandler('bets')
+export default handler
```

Comment:
If we remove the vercel cache and forward to our api, I think that can actually cost us a lot: vercel will treat each call as non-cached and charge us for each. (Whereas right now it doesn't charge us if it hits the vercel cache.)

I think we need to keep all the same caches as before for all the vercel v0 api.